sklearn/linear_model/_stochastic_gradient.py

```diff
@@ -22,6 +23,9 @@
 from ..utils.extmath import safe_sparse_dot
 from ..utils.multiclass import _check_partial_fit_first_call
 from ..utils.validation import check_is_fitted, _check_sample_weight
+from ..utils._param_validation import Interval
+from ..utils._param_validation import StrOptions
+from ..utils._param_validation import Hidden
```

Comment:
```suggestion
from ..utils._param_validation import Hidden, Interval, StrOptions
```

Reply:
I'd rather on isort https://github.com/scikit-learn/scikit-learn/pull/23362 :D

Reply:
It is to be consistent with the other imports in the other files :)